src/bootstrap.php

```diff
@@ -1,7 +1,12 @@
 <?php
+$dbConnectionPath = __DIR__ . '/../src/Vox/bootstrap.db.connection.php';
+
 return array(
     'import' => array(
         (array) require __DIR__ . '/../src/Vox/bootstrap.php',
-        (array) require __DIR__ . '/../src/Console/bootstrap.php'
+        (array) require __DIR__ . '/../src/Vox/bootstrap.db.php',
+        is_file($dbConnectionPath) ? (array) require $dbConnectionPath : [],
```

Comment:
`file_exists` would return `true` also for directories also `is_file` used to be faster